Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md

```diff
@@ -0,0 +1,70 @@
+# Users on Umbraco Cloud
+
+On Umbraco Cloud, users work in almost the same way as on a normal installation of Umbraco; however, there are a few more settings available for the backoffice users on Umbraco Cloud.
+
+In this article, we will show how users work, as well as explain the different settings for users on Umbraco Cloud.
+
+## Adding users on Umbraco Cloud
+
+There are two ways of adding a user to your backoffice on Umbraco Cloud.
+
+You can add them as a [Team member](/Umbraco-Cloud/Set-Up/Team-Members/) from the project portal on Umbraco Cloud.
+
+By default, when added to the project as a team member, they will be added to the backoffice as administrators.
+
+Users can also be invited directly from the backoffice of your Umbraco Cloud project, from where you can give them different permissions.
+
+Check out the [Users article](/Getting-Started/Data/Users/) for an in-depth explanation about Umbraco users in general.
+
+:::note
```

Comment:
Should we add to this note, that if they're added as Team Members then they will be added to ALL environments?